src/test/codewhisperer/service/keyStrokeHandler.test.ts

```diff
@@ -100,19 +98,6 @@ describe('keyStrokeHandler', function () {
             assert.ok(!invokeSpy.called)
         })
 
-        it('Should not call invokeAutomatedTrigger if previous text input is within 2 seconds and it is not a specialcharacter trigger \n', async function () {
```

Comment:
We're removing a decent number of tests without any new tests to cover for them. Are these deleted tests covered by something else? If not, is there any way to extract out some of the logic in the `idleTriggerTimer` callback and test the same/similar behavior?

Reply:
added tests